adyenv6core/src/com/adyen/v6/factory/AdyenRequestFactory.java

```diff
@@ -227,34 +224,36 @@ private void updatePaymentRequestForCC(PaymentsRequest paymentsRequest, CartData
         String encryptedExpiryMonth = cartData.getAdyenEncryptedExpiryMonth();
         String encryptedExpiryYear = cartData.getAdyenEncryptedExpiryYear();
 
-        if (!StringUtils.isEmpty(encryptedCardNumber) && !StringUtils.isEmpty(encryptedExpiryMonth) && !StringUtils.isEmpty(encryptedExpiryYear)) {
+        if (! StringUtils.isEmpty(encryptedCardNumber) && ! StringUtils.isEmpty(encryptedExpiryMonth) && ! StringUtils.isEmpty(encryptedExpiryYear)) {
```

Comment:
Is it an autoformatter - maybe it's just me but like better without space :D

Reply:
indeed autoformatter